### PR TITLE
BUG Add Darkmod style for popover

### PR DESCRIPTION
**Pull request by maxime-rainville**
Fixes https://github.com/silverstripe/silverstripe-login-forms/issues/96

That's what it look like

image
image

### CODE REVIEW

File: client/src/styles/login-form.scss [hunk 1]
```diff
@@ -207,7 +207,23 @@ input[type="checkbox"] {
...
 }
 
-.popover .popover-header + .popover-body {
-  padding: 12px 15px;
-  font-size: 13px;
+.popover {
+  background-color: var(--color-form-background);
+  border-color: var(--color-field-border);
+
+  &.bs-popover-auto[x-placement^="top"] > .arrow::before,
```
Comment (scott1702) [inline client/src/styles/login-form.scss]: In some cases the popover could be placed at the bottom 

image

**Comment by clarkepaul** [comment]
Colours look good!

File: client/src/styles/login-form.scss (hunk 1 above)
Comment (maxime-rainville) [reply client/src/styles/login-form.scss]: This should fix it https://github.com/silverstripe/silverstripe-login-forms/pull/98/commits/acc1b4b4972c44c5bce91f5f7143e0057f0e2a37

**Review by scott1702 — APPROVED** [review]

**Comment by bergice** [comment]
image

Looks alright. @clarkepaul do you have any requests? I'm thinking maybe it should stand out more from the rest of the form behind it.

**Comment by maxime-rainville** [comment]
Yeah ... I think the light mode popup has a shadow which make it stand out a bit more over tho other content.

**Comment by clarkepaul** [comment]
I think its all good :). With movement and the fact you have triggered the popup with your cursor right next to it the users attention is already focused there.

**Pull request merged**